src/main/java/com/icthh/xm/ms/entity/service/VoteService.java

```diff
@@ -41,7 +44,10 @@ public Vote save(Vote vote) {
                                                               vote.getId(),
                                                               voteRepository,
                                                               Vote::setEntryDate,
+
                                                               Vote::getEntryDate);
+        XmEntity entity = xmEntityRepository.getOne(vote.getXmEntity().getId());
```

Comment:
You are using a different style. In some cases direct and without `XmEntity entity`. In this case with `XmEntity entity`. Why?